src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -323,6 +325,18 @@ class WorkspaceMembersPage extends React.Component {
             || this.isKeywordMatch(member.firstName, searchValue)
             || this.isKeywordMatch(member.lastName, searchValue));
 
+        data = _.reject(data, (member) => {
+            // If this policy is owned by Expensify then show all support (expensify.com or team.expensify.com) emails
+            if (PolicyUtils.isExpensifyTeam(lodashGet(this.props.policy, 'owner'))) {
+                return;
+            }
```

Comment:
This caused a "bug" https://github.com/Expensify/App/issues/17252. The root cause was internal, just noting here for reference. If we delete the workspace the owner would be `undefined` and the `PolicyUtils.isExpensifyTeam` will crash.